src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
  * Deprecate `ContainerAwareInterface` and `ContainerAwareTrait`, use dependency injection instead
  * Add `defined` env var processor that returns `true` for defined and neither null nor empty env vars
  * Add `#[AutowireLocator]` and `#[AutowireIterator]` attributes
+ * Add `#[AutowireInline]` attribute
```

Comment:
should go in a new 7.1 section

Reply:
yes please, new features go on 7.1 now

Reply:
I've rebased this onto 7.1 thanks for letting me know :-)